mne_bids/dig.py

```diff
@@ -158,7 +158,8 @@ def _electrodes_tsv(raw, fname, kind, overwrite=False, verbose=True):
     fname : str
         Filename to save the electrodes.tsv to.
     kind : str
-        Type of the data as in ALLOWED_KINDS. For iEEG, requires size.
+        Type of the data. One of ``'meg'``, ``'eeg'``, ``'ieeg'``.
+        For iEEG, requires size.
```

Comment:
what does this mean?

Reply:
It encodes the modality. I think `'mri'` is missing here

Reply:
These are just types of coordinate data: so eeg, meg, ieeg are the only ones with `electrodes.tsv`

Reply:
> These are just types of coordinate data

right. Sorry, I was having a little trouble with some changes GH has made to the diff views.

So: this is fine with me.